go-controller/pkg/ovn/ovn.go

```diff
@@ -85,6 +86,8 @@ type Controller struct {
 	joinSubnetAllocator     *subnetallocator.SubnetAllocator
 	nodeLocalNatIPAllocator *ipallocator.Range
 
+	hoMaster *hocontroller.MasterController
```

Comment:
Why does this need to be a member?

Reply:
@dave-tucker because I move it's Run() from StartClusterMaster() to the Controller's Run() function, so we need to keep that pointer across function calls.